web/themes/joinup/prototype/sass/components/_form.sass

```diff
@@ -98,5 +99,33 @@
 .form__description--checkbox
   margin-left: 25px
 
+// Dependencies
+// --------------------------------------------------
+.form__actions
```

Comment:
I split the styles between `form` and `form-table` because I did not want to keep these styles in one place because it was completely different components but with the same markup. If you would like I can move all the styling to `_drupal-hacks` file, there is no problem.